src/main/java/org/isf/telemetry/model/Telemetry.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2022 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+ *
+ * Open Hospital is a free and open source software for healthcare data management.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.isf.telemetry.model;
+
+import java.lang.reflect.Type;
+import java.time.LocalDateTime;
+import java.util.Map;
+
+import javax.persistence.Column;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+/**
+ * @author Andrei
+ *
+ */
+@Entity
+@Table(name = "OH_TELEMETRY")
+public class Telemetry {
+
+	@EmbeddedId
+	private TelemetryId id;
+
+	// necessary for daemon. Used to understand if daemon is active. TRUE | FALSE | NULL
+	@Column(name = "TEL_ACTIVE")
+	private Boolean active;
+
+	@Column(name = "TEL_CONSENT")
+	private String consentData;
+
+	@Column(name = "TEL_INFO")
+	private String info;
+
+	@Column(name = "TEL_SENT_TIME")
+	private LocalDateTime sentTimestamp;
+
+	@Column(name = "TEL_OPTIN_DATE")
+	private LocalDateTime optinDate;
+
+	@Column(name = "TEL_OPTOUT_DATE")
+	private LocalDateTime optoutDate;
+
+	@Transient
+	private volatile int hashCode = 0;
```

Comment:
Setting value to zero is redundant as that is the default value.

Reply:
yes, done, thank you